packages/crayons-core/src/components/select/select.tsx

```diff
@@ -229,6 +233,17 @@ export class Select {
    * Describes the select's boundary HTMLElement
    */
   @Prop() boundary: HTMLElement;
+  /**
+   * Props to be passed for creatable select
+   * isCreatable: boolean - If true, select accepts user input that are not present as options and add them as options
+   * validateNewOption: (value) => boolean - If passed, this function will determine the error state for every new option entered
```

Comment:
can you call out, what does it imply if `true` is returned or `false` is returned.